src/task2/task2.js

```diff
@@ -0,0 +1,8 @@
+    function min(n, m)
+    {
+        if (n===m) return "equal"
```

Comment:
also:
```
if (n===m) {
    ....
} else if (x===y) {
    ....
} else {
    ......
}
```

Reply:
No joke, my eyes are bleeding)
It's all in the topics, please don't read it through your fingers

Reply:
sorry I made u suffer. I'll try to pull my weight in the future

Reply:
![image](https://user-images.githubusercontent.com/44597516/214944101-f4a3089b-b34e-45b7-a123-0cbfae640b51.png)
And here we are again)

Reply:
and **always** end return value/expression with a semicolon

Reply:
soooorry, I didn't get it back then, transparent as your comments might have been